source/CMakeLists.txt

```diff
@@ -226,6 +226,9 @@ target_sources(EndlessSkyLib PRIVATE
 	OutlineShader.h
 	Panel.cpp
 	Panel.h
+	Parallel.cpp
+	Parallel.h
+	PartiallyGuarded.h
```

Comment:
Should there be a `PartiallyGuarded.cpp`?

Reply:
That file is header-only because it only contains templates. (Don't worry about it.)